### DISCUSSION TITLE
Players cant build without a claim

### DISCUSSION THREAD

**Discussion by gold0s** [GriefPrevention/GriefPrevention#1362] [Q&A]
so I've been trying to make this work but i cant figure it out. it just wont let you build or break blocks without making a claim first the game mode is in survival i have no other plugins. help?

**Comment by bigpresh** [comment]
Make sure Claims.mode is set to Survival, not Creative or SurvivalRequiringClaims:
https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#limiting-which-worlds-griefprevention-runs-on
